pkg/diskmaker/controllers/lv/add.go

```diff
@@ -142,6 +141,8 @@ func handlePVChange(runtimeConfig *provCommon.RuntimeConfig, pv *corev1.Persiste
 		// delayed reconcile so that the cleanup tracker has time to mark the PV cleaned up
 		time.Sleep(time.Second * 10)
 		q.Add(reconcile.Request{NamespacedName: types.NamespacedName{Name: ownerName, Namespace: ownerNamespace}})
+		time.Sleep(time.Second * 120)
+		q.Add(reconcile.Request{NamespacedName: types.NamespacedName{Name: ownerName, Namespace: ownerNamespace}})
```

Comment:
Not sure how controller-runtime works, do you really block informer callback for 2 minutes? Or does controller-runtime call each callback in its own goroutine? Blocking the whole informer for 2 minutes is wrong.

Reply:
Removed this, turned out to be necessary with the fixed watches.